docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-BUILD.xml

```diff
@@ -903,25 +925,86 @@ BUILD(FilterDsLib1);
     without decompression. The original index compression implementation
     decompresses the rows when they are read from disk.</para>
 
+    <para>The inplace index compression format (introduced in versions 9.6.90,
+    9.8.66, and 9.10.12 9.2.0 or later) improves compression and reduces build
+    time. These formats require an engine that supports it. In other words,
+    <emphasis role="bold">if you build an index using the lz4s or lz4shc
+    formats, you must use a platform later than 9.6.90, 9.8.66, and 9.10.12 to
+    read those indexes. </emphasis></para>
+
+    <para>If you attempt to read an index with the inplace compression format
+    on a system that does not support it, you will receive an error
+    message.</para>
+
     <para>Because the branch nodes can be searched without decompression more
     branch nodes fit into memory which can improve search performance. The lz4
     compression used for the payload is significantly faster at decompressing
-    leaf pages than the previous LZW compression.</para>
+    leaf pages than the previous LZW compression. Whether performance is
+    better with lz4hc (a high-compression variant of lz4) on the payload
+    fields depends on the access characteristics of the data and how much of
+    the index is cached in memory.</para>
 
-    <para>Whether performance is better with lz4hc (a high-compression variant
-    of lz4) on the payload fields depends on the access characteristics of the
-    data and how much of the index is cached in memory.</para>
+    <para><emphasis role="bold">Compression Levels :</emphasis></para>
 
-    <para>If you attempt to read an index with the inplace compression format
```

Comment:
This paragraph is still true.  Possibly should be extended with "or an inplace compression format"